plugins/destination/postgresql/client/list_tables.go

```diff
@@ -102,17 +102,14 @@ func (c *Client) listTables(ctx context.Context, include, exclude []string) (sch
 	return tables, nil
 }
 
-func (c *Client) whereClause(include, exclude []string) string {
-	if len(include) == 0 && len(exclude) == 0 {
+func (c *Client) whereClause(include []string) string {
+	if len(include) == 0 {
 		return ""
 	}
 	var where string
 	if len(include) > 0 {
 		where = fmt.Sprintf("AND pg_class.relname IN (%s)", c.inClause(include))
 	}
-	if len(exclude) > 0 {
-		where = fmt.Sprintf("AND pg_class.relname NOT IN (%s)", c.inClause(exclude))
```

Comment:
I think there was a bug here. There should be a space between the include and exclude `AND` clauses